client/src/screens/Map/index.tsx

```diff
@@ -53,11 +50,22 @@ export const MapScreen: NavigationScreenComponent<MapScreenParams> = ({navigatio
 
     const {data, loading, error} = useQuery<Types.Query>(GET_POIS, {variables: filter});
     useEffect(_.partial(messageBox.error, error), [error]);
-    const pois = data && data.pois;
+    const pois = ((data && data.pois) || []) as Types.Poi[];
 
+    const [currentRegion, setCurrentRegion] = useState(initialMapRegion);
```

Comment:
I think it would be nice to have `initialMapRegion` equal to the user's current location if he/she is situated in Ryazan'

Reply:
i'd say we show people how to get to ryazan if they aren't :D